webpack/rules/css.js

```diff
@@ -30,11 +32,18 @@ module.exports = ({ production = false, browser = false } = {}) => {
         minimize: true,
         importLoaders: 1
       }
-    },
-    {
-      loader: "sass-loader"
-    },
-    {
+    }, {
+      loader: 'postcss-loader',
+      options: {
+        ident: 'postcss',
+        plugins: [
+          postcssPresetEnv({ stage: 2, browsers: 'last 2 versions' }),
```

Comment:
Not sure about IE11, I played around here to find a good level of support https://preset-env.cssdb.org/playground. I think we should keep all configuration within one file if possible (webpack/rules/css) instead of splitting it up. 

Autoprefixer is included in preset-env so we can exclude it. Please add the object-fit plugin, we need to start using this and srcsets more. Do we need cssnano, it is not already being minified?